src/main/java/seedu/address/ui/PersonDetail.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Tab;
+import javafx.scene.layout.Region;
+
+public class PersonDetail extends UiPart<Region> {
+    private static final String FXML = "PersonDetail.fxml";
+
+    @FXML
+    private Tab skill;
+
+    @FXML
+    private Tab academic;
+
+    @FXML
+    private Tab interview;
+
+    @FXML
+    private Tab statement;
+
+
+
+    public PersonDetail() {
+        super(FXML);
+    }
+
+    /*  public PersonDetail(Person person) {
+        super(FXML);
+        setSkillTab(person.getSkill)
+        setInterviewTab(person.getInterview)
+        setStatementTab(person.getStatement)
+        setAcademicTab(person.getAcademic)
+    }
+
+
+    void setSkillTab(String text) {
```

Comment:
skill could be merged with academic, so we can have interview, academic, statement